client/src/components/about1/Demo.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import ReactPlayer from "react-player";
+
+function Demo() {
+  return (
+    <div className="flex items-start  pl-7 pt-5 w-full bg-cover bg-pink-950 min-h-screen">
+      <div className=""></div>
+      <div className="hero-content text-center text-neutral-content">
+        <div className="max-w-lx">
+          <h1 className="mb-2 text-2xl text-left font-bold">
+            How to use the app?
+          </h1>
+
+          <div>
+            <div className="w-[60rem] text-xl text-justify">
+              <div className="">
+                <p>
+                  In the below Video you will discover: <br />
+                  <h2>Part 1</h2>
+                  <span>
+                    How to add CYF topic to your dashboard?. How to discover the
+                    topic desorption?. & How to review the topic?{" "}
+                  </span>
+                  <h2>Part 2</h2>
+                  <span>
+                    How to create new topic from CYF modules?. How to add it
+                    into your dashboard?. & How to review the topic?{" "}
+                  </span>
+                </p>
+              </div>
+            </div>
+          </div>
```

Comment:
This section doesn't bring much to the user and it doesn't read easily 
Please, consider removing it completely or swapping it for something more valuable